File: examples/stock/price/node/getAStockPrice.js
const { StockService } = require("micro-js-client/stock");

// Get the last price for a given stock ticker
async function getAstockPrice() {
  let stockService = new StockService(process.env.MICRO_API_TOKEN);
  let rsp = await stockService.price({
    symbol: "AAPL",
  });
  console.log(rsp);
}

getAstockPrice();

File: examples/go/stock/price/getAStockPrice.go
package example

import (
	"fmt"
	"os"

	"github.com/micro/services/clients/go/stock"
)

// Get the last price for a given stock ticker
func GetAstockPrice() {
	stockService := stock.NewStockService(os.Getenv("MICRO_API_TOKEN"))
	rsp, err := stockService.Price(&stock.PriceRequest{
		Symbol: "AAPL",
	})
	fmt.Println(rsp, err)
}
